app/helpers/login_helper.rb

```diff
@@ -0,0 +1,16 @@
+module LoginHelper
+  def login_props
+    alert_type = flash_inline.keys[0]
+    {
+      token: form_authenticity_token,
+      version: SETTINGS[:version].version,
+      caption: Setting[:login_text],
+      alertType: alert_type,
+      alertMessage: flash_inline[alert_type]
```

Comment:
this assumes there is only one message? I don't expect more but....?

Reply:
passing the whole `flash_inline object` now,
blocking PR at patternfly: https://github.com/patternfly/patternfly-react/pull/1974
for supporting multiple alerts.